be/src/olap/rowset/beta_rowset_reader.h

```diff
@@ -48,7 +48,7 @@ class BetaRowsetReader : public RowsetReader {
 
     RowsetSharedPtr rowset() override { return std::dynamic_pointer_cast<Rowset>(_rowset); }
 
-    int64_t filtered_rows() override { return _stats->rows_del_filtered; }
+    int64_t filtered_rows() override { return _stats->rows_del_filtered + _stats->rows_conditions_filtered; }
```

Comment:
Add a comment for this method to notify that this method will be used for validation of schema change task, should be modified carefully.

Reply:
Related comment have been added, I will be familiar with the development process again.